vio/viosslfactories.c

```diff
@@ -182,6 +182,13 @@ static int wolfssl_send(WOLFSSL* ssl, char* buf, int sz, void* vio)
 {
   return (int)vio_write((Vio *)vio, (unsigned char*)buf, sz);
 }
+
+#ifndef TLS1_3_VERSION
+#define TLS1_3_VERSION
+#endif
+#ifndef TLS1_2_VERSION
+#define TLS1_2_VERSION
```

Comment:
Do we need that?

Reply:
though it was a cleaner than `defined(TLS1_2_VERSION) || defined(HAVE_WOLFSSL)` below however these are correctly defined in `extra/wolfssl/wolfssl/wolfssl/openssl/{tls1,ssl}.h`. I will remove them.

Also Vlad, was there a reason FP_MAX_BITS to 16k was conditional on WOLFSSL_FASTMATH? Still trying to fix main.ssl_8k_key test for TLSv1.3 per comments from wolfssl upstream. There's probably something else other than just defining it as I'm still getting segfaults.

Reply:
I forgot most of details, but IIRC, FP_MAX_BITS 16K was to please this unrealistic ssl_8k_key test. Nobody in his right mind would allow keys that big, I guess. I think this fastmath was stack based, and alternative was heap based,  but I frankly do not remember very much about that.

Reply:
Thanks. I'll look more closely at that before I finalise this. Still trying to appease the 8k test in tls1.3, the test that keeps on giving :-)